login-github/index.js

```diff
@@ -0,0 +1,56 @@
+const puppeteer = require("puppeteer");
+const readline = require("readline");
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let userName = "";
+let password = "";
+
+const ask_user_name = () => {
+  return new Promise((resolve, reject) => {
+    rl.question("What is your user name ? \n ", (answer) => {
+      userName = answer;
+      console.log(`your your user name is ${userName}\n`);
```

Comment:
shouldn't it be ?
```js
console.log(`your user name is ${userName}\n`);
```
same is with password

Reply:
Thanks for reviewing.